src/use-swr.ts

```diff
@@ -175,7 +175,8 @@ export const useSWRHandler = <Data = any, Error = any>(
 
           // Start the request and keep the timestamp.
           CONCURRENT_PROMISES_TS[key] = startAt = getTimestamp()
-          newData = await (CONCURRENT_PROMISES[key] = fn.apply(fn, fnArgs))
+          // @ts-ignore FIXME LATER
```

Comment:
the dependency of typescript eslint `tsutils` didn't resolve the right version of tslib which lead to the error.
you can add a resolution for tslib to 2.3.0 in root top package.json. but might be hacky.

I submit a PR to it but didn't get merged...

Reply:
I separate some upgration changes to https://github.com/vercel/swr/pull/1486